docker/README.md

```diff
@@ -0,0 +1,109 @@
+<!---
+Licensed under Creative Commons Attribution 4.0 International License
+https://creativecommons.org/licenses/by/4.0/
+--->
+
+# Docker Tools and Usages #
+
+This directory contains scripts useful for building and running a
+container with the PDO contracts. For the most part, this directory
+could be copied to any host (even without PDO otherwise installed) to
+build, configure, and execute PDO contracts code.
+Also note that the makefile assumes you have already built all containers from the `private-data-objects` repository.  If not, just run `make repository; make -C repository/private-data-objects/docker`.
+For clarity the scripts in the directory are intended to serve two
+purposes. The first is to provide the basic instructions for building
+images for running the various contract families in the PDO contracts
+repository. The images may be used for a variety of uses including
+demonstration, testing and interacting with PDO contracts. The second
+purpose is to provide a means for clean, reproducible testing of the
+contracts and the Jupyter notebooks. Fully automated testing
+(e.g. automated execution of all contract family tests) is provided
+through the `test` target in the `Makefile` (e.g. run `make test` to
+perform the automated tests). Interactive testing of the Jupyter
+notebooks is provided through the `test_jupyter` target. That target
+will create and run a container with a Jupyter server running inside.
```

Comment:
From a legibility perspective, this entire section of the doc appears as one big block of text on the GitHub UI. Can you please break up these instructions into a couple paragraphs or a list of steps? Just makes it much easier to consume quickly.